contracts/BlankArt.sol

```diff
@@ -114,7 +123,7 @@ contract BlankArt is ERC721, EIP712, ERC721URIStorage {
         else
             _base = _baseURIs[_baseURIs.length -1];
 
-        return string(abi.encodePacked(_base, Strings.toString(tokenId)));
+        return string(abi.encodePacked(_base, Strings.toString(tokenId), '.json'));
```

Comment:
Added the .json extension, as it seems to require it for any tokens above 1, but let me know if there are other approaches here.